Common/POpen.cpp

```diff
@@ -85,7 +85,7 @@ POpen::POpen(const std::string& cmd) : m_Command(cmd)
 	// before we can be confident that the process is running ok.
 	DWORD msWait = 10000; // Wait at most 10 seconds 
```

Comment:
Since we use INFINITE hardcoded, let's remove msWait (or use msWait = INFINITE)

Reply:
here we are going to call WaitForInputIdle() which should just return as soon as the child process is Idle, so 10s should be plenty enough

reading the rest, I suspect that perhaps it's not working for calling the P4 MFA application, that it's never idle, eg always doing some active loop? (I cannot test it myself at this stage, I don't have VS2015 installed (yet) because it broke Unreal...)

Reply:
A little refactor two clear things up has been done. Will try to accomplish a more deep refactor for P4SpecCommand asap